db/migrate/20180408135739_create_searchgov_domains.rb

```diff
@@ -0,0 +1,15 @@
+class CreateSearchgovDomains < ActiveRecord::Migration
+  def change
+    create_table :searchgov_domains do |t|
+      t.string :domain, null: false
+      t.boolean :clean_urls, null: false, default: true
+      t.string :status
+      t.integer :urls_count, null: false, default: 0
+      t.integer :unfetched_urls_count, null: false, default: 0
+
+      t.timestamps null: false
+    end
+
+    add_index :searchgov_domains, :domain, length: 100, unique: true
```

Comment:
Is there any reason why this shouldn't use `length: 255` since the`domain` column is that big?

Reply:
Yes, MySQL barfs when you try to create a unique index on 255 char. I considered a) making the domains column only 100 char, but *technically* a domain name can be up to 255 char. I figured that the chances of our domains even hitting 100 char are slim enough that I'm not worried about false positives on a unique check for two different 100+ char domains with the same first 100 characters.

Reply:
Ah... that makes sense. I had checked the index on `searchgov_urls.url` and saw that it was 255 characters, but that index isn't unique. So it wasn't right for me to use that as a comparison 👍.